scripts/deployment/restart_server.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o xtrace
+
+# https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad
+
+# This script is run on each Open Library host to finish a new deployment.
+
+cd /opt/openlibrary
+export HOSTNAME="${HOSTNAME:-$HOST}"
+export COMPOSE_FILE="docker-compose.yml:docker-compose.production.yml"
+# WARNING! Moment of downtime 😬 
+docker-compose down
+docker volume rm openlibrary_ol-vendor openlibrary_ol-build openlibrary_ol-nodemodules
+if [[ $HOSTNAME == ol-covers0.* ]]; then
+    docker-compose up -d --scale covers=2 covers_nginx memcached
+elif [[ $HOSTNAME == ol-home0.* ]]; then
+    docker-compose up -d --no-deps infobase infobase_nginx affiliate-server  # cronjobs importbot solr-updater
+else  # start a web node
```

Comment:
Don't use else use `elif` for ol-web...; have an else case that throws an error, just in case the wrong hostname is passed in.